packages/graphback-core/src/runtime/CRUDService.ts

```diff
@@ -55,7 +55,8 @@ export class CRUDService<Type = any> implements GraphbackCRUDService<Type>  {
       const topic = this.subscriptionTopicMapping(GraphbackOperationType.CREATE, this.model.graphqlType.name);
       //TODO use subscription name mapping
       const payload = this.buildEventPayload('new', result);
-      await this.pubSub.publish(topic, payload);
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      this.pubSub.publish(topic, payload).finally(() => {});
```

Comment:
ok. Havent's seen that. Why finnally is empty? Makes no sense

Reply:
We can surely add logs, I was just too lazy to add them.

Reply:
I've added a console.error on this, so that users get's to see what went wrong.